src/Application/Interfaces/IRepositories/IGenericRepository.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Linq.Expressions;
 
-namespace Application.Interfaces
+namespace Application.Interfaces.IRepositories
 {
     public interface IGenericRepository<T> where T : class
```

Comment:
you also should add default value for tracking param in GetAll and Search methods

Reply:
i added GenericRepository to IRepository

Reply:
We added default values in entity reposisories for tracking params, and in Services we changed GetAll to GetPagedAsync